update_and_render.sh

```diff
@@ -5,7 +5,15 @@ curl -o rpc.proto -s https://raw.githubusercontent.com/lightningnetwork/lnd/mast
 # Generate the rpc.json file from rpc.proto, so that generate_slate_docs.py can parse it
 protoc -I. -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --doc_out=json,rpc.json:. rpc.proto
 
-# Generate the new docs
+# Update lncli to the respective commit.
+pushd $GOPATH/src/github.com/lightningnetwork/lnd
+git pull
+git reset --hard $commit
```

Comment:
Why not a `git checkout`?

Reply:
Tried this locally just now, and had to manually approve a merge in vim. Will need to ensure that the update bot doesn't get stuck on something similar.

Reply:
Fixed.